### PR TITLE
Added new page for Developer Health resources.  Starting with "Burnout"

### PR DESCRIPTION
**Pull request by chaos986**
## Linked Issue

#673 

## Description

I made a new page and pasted in text from slack and videos from YouTube. 

I am still new to Markdown and React.  

I previewed nothing. 

This could break prod. 

## Code of Conduct

> By submitting this pull request, you agree to follow our [Code of Conduct](https://virtualcoffee.io/code-of-conduct/)

### CODE REVIEW

File: app/routes/__frontend/resources/developer-health/index.mdx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+---
+
+import TextContainer from '~/components/content/TextContainer';
+import FileIndex from '~/components/content/FileIndex';
+
+<TextContainer background="light" showBackToTopLink={false}>
+
+## Developer Health
+
+#Burnout
```
Comment (JoeKarow) [inline app/routes/__frontend/resources/developer-health/index.mdx:20]: Need a space between '#' & "Burnout"

**Review by JoeKarow — CHANGES_REQUESTED** [review]

File: app/routes/__frontend/resources/developer-health/index.mdx (hunk 1 above)
Comment (chaos986) [reply app/routes/__frontend/resources/developer-health/index.mdx]: done

**Comment by BekahHW** [comment]
This is looking good, @chaos986. I have a couple of comments about structuring the page.

Let's start by changing Burnout to "Managing burnout". 

I've updated the content to read more like a blog post. I should've put this in the issue, so that's on me. I think the section will work out to be more like a collection of guides.

[Here's the updated version.](https://docs.google.com/document/d/15GyVHEkWPEWNAHitgLwow-C4h-1JW9_6no_9O5T9jXU/edit?usp=sharing)

Below that, add a section heading for "Video Resources".

Let me know if you have questions.

**Comment by BekahHW** [comment]
@chaos986 wanted to check-in here. How are things going? Do you need support?

**Pull request merged**